From 81897756008e8a4dc74c16614323bd42e147d972 Mon Sep 17 00:00:00 2001
From: Darshit Shah <darnir@gmail.com>
Date: Sat, 29 Mar 2014 11:48:35 +0100
Subject: [PATCH] Custom statusline

---
 plugin/ctags.vim | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/plugin/ctags.vim b/plugin/ctags.vim
index 4a779b7..95b0c31 100644
--- a/plugin/ctags.vim
+++ b/plugin/ctags.vim
@@ -321,9 +321,9 @@ function! s:SetTagDisplay()
 	let w:tag_name = l:tag_name
 	let s:ruler = &ruler
 	if &ruler
-	    let &statusline='%<%f %(%h%m%r %)%=%{TagName()} %-15.15(%l,%c%V%)%P'
-					" Equivalent to default status
-					" line with 'ruler' set:
+            let &statusline="%#VertSplit#%F\ [%{strlen(&fenc)?&fenc:'none'},%{&ff}]%h%m%r%y%{fugitive#statusline()}%=%{TagName()} %c,%l/%L\ %P"
+                                        " Equivalent to default status line with
+                                        " 'ruler' set:
 					"
 					" '%<%f %h%m%r%=%-15.15(%l,%c%V%)%P'
 	else
-- 
1.9.1

